cylc/flow/cfgspec/workflow.py

```diff
@@ -626,21 +626,33 @@ def get_script_common_text(this: str, example: Optional[str] = None):
             365 day (never a leap year) and 366 day (always a leap year).
         ''')
         Conf('runahead limit', VDR.V_STRING, 'P4', desc='''
-            The scheduler runahead limit determines how many consecutive cycle
-            points can be active at once. The base point of the runahead
-            calculation is the lowest-valued point with :term:`active` or
-            :term:`incomplete` tasks present.
+            The runahead limit prevents a workflow from getting too far ahead
+            of the oldest active cycle.
 
-            An integer interval value of ``Pn`` allows up to ``n+1`` cycle
-            points (including the base point) to be active at once.
+            A cycle is considered to be active if it contains any
+            :term:`active` or :term:`incomplete` tasks.
```

Comment:
See https://github.com/cylc/cylc-doc/pull/660. We need a consistent way to describe this.